src/compiler/transformers/add-static-style.ts

```diff
@@ -92,11 +97,11 @@ const getMultipleModeStyle = (
     }
   });
 
-  return ts.createObjectLiteral(styleModes, true);
+  return ts.factory.createObjectLiteralExpression(styleModes, true);
 };
 
 const createPropertyAssignment = (mode: string, initializer: ts.Expression) => {
-  const node = ts.createPropertyAssignment(mode, initializer);
+  const node = ts.factory.createPropertyAssignment(mode, initializer);
   return node;
 };
```

Comment:
this function strikes me as quite pointless, thoughts on removing it?

Reply:
I'd condense it

Reply:
On one hand, I agree.

On the other, I see how the level of abstraction _could_ help here. With wrappers like these, in theory we'd only ever need to update in once place should the TS API change again (rather than hunt for `/ts\.[^factory]/` like we're gonna do). 

Butttttt I don't think we're in a good place to take on _that_ work - so I say let's just rm it